MvvmCross.Forms/Platform/Android/Views/MvxFormsApplicationActivity.cs

```diff
@@ -48,6 +49,8 @@ public IMvxViewModel ViewModel
             }
         }
 
+        private MvxNotifyTask _startTaskNotifier;
```

Comment:
This member isn't used anywhere.

Reply:
Thanks. Will clean it up now that it’s no longer used